### PR TITLE
Process powerbalance parameters

### PR DESCRIPTION
**Pull request by je-cook**
## Description

Creates mappings to currently non existent bluemira parameters.

## Interface Changes

If you've had to update an interface or introduce a new interface as part of your change then let us know here.

~This is based off #748 and that needs to be merged first~

## Checklist

I confirm that I have completed the following checks:

- [x] Tests run locally and pass `pytest tests --reactor`
- [x] Code quality checks run locally and pass `flake8` and `black .`
- [x] Documentation built locally and checked `sphinx-build -W documentation/source documentation/build`

### CODE REVIEW

File: bluemira/codes/process/mapping.py [hunk 1]
```diff
@@ -86,6 +135,13 @@
     "r_vv_ob_in": ("r_vv_ob_in", "m"),
     "r_tf_out_centre": ("r_tf_outboard_mid", "m"),
     "g_vv_ts": ("gapds", "m"),
+    "TF_res_bus": ("tfbusres", "m"),
+    "TF_res_tot": ("ztot", "ohm"),
+    "TF_E_tot": ("estotftgj", "GJ"),
+    "TF_respc_ob": ("tflegres", "ohm"),
+    "TF_currpt_ob": ("cpttf", "A"),
+    "P_bd_in": ("pinjmw", "MW"),
+    "condrad_cryo_heat": ("qss/1.0D6", "MW"),
```
Comment (CoronelBuendia) [inline bluemira/codes/process/mapping.py:144]: Not sure about the others, shouldn't these go in config.py too? Would help to have full parameters for them

**Review by CoronelBuendia — CHANGES_REQUESTED** [review]

File: bluemira/codes/process/mapping.py (hunk 1 above)
Comment (je-cook) [reply bluemira/codes/process/mapping.py:144]: yeah and config_schema.py, I just wanted to have the names sorted before I did that

File: bluemira/codes/process/mapping.py [hunk 2]
```diff
@@ -86,6 +135,13 @@
     "r_vv_ob_in": ("r_vv_ob_in", "m"),
     "r_tf_out_centre": ("r_tf_outboard_mid", "m"),
     "g_vv_ts": ("gapds", "m"),
+    "TF_res_bus": ("tfbusres", "m"),
+    "TF_res_tot": ("ztot", "ohm"),
+    "TF_E_tot": ("estotftgj", "GJ"),
```
[the remark this answers is not in the document]
Comment (je-cook) [reply bluemira/codes/process/mapping.py:140]: The dict descriptions says: ''total stored energy in the toroidal field (GJ)"

**Review by CoronelBuendia — APPROVED** [review]
I fear we're going to end up writing a Python interface to PROCESS if we keep this up.. Understand these are necessary for the time being!

**Comment by je-cook** [comment]
> I fear we're going to end up writing a Python interface to PROCESS if we keep this up.. Understand these are necessary for the time being!

Yeah I know, hopefully it will just speed up the conversion as they can strip this out and use it directly

File: bluemira/codes/plasmod/api.py [hunk 3]
```diff
@@ -336,20 +336,23 @@ class Setup(interface.Setup):
...
+    def __init__(self, parent, *args, **kwargs):
         super().__init__(parent, *args, **kwargs)
 
-        self._problem_settings = problem_settings if problem_settings is not None else {}
         self.input_file = "plasmod_input.dat"
         self.output_file = "plasmod_outputs.dat"
         self.profiles_file = "plasmod_profiles.dat"
-        self.io_manager = Inputs({**self.get_new_inputs(), **self._problem_settings})
+        self.io_manager = Inputs(
+            {**self._get_new_inputs(), **self.parent.problem_settings}
```
Comment (hsaunders1904) [inline bluemira/codes/plasmod/api.py:346]: Wasn't this changed to a public method in #748?

File: bluemira/codes/process/setup.py [hunk 4]
```diff
@@ -84,9 +104,13 @@ def write_indat(self, use_bp_inputs=True):
             )
 
         if use_bp_inputs is True:
-            _inputs = self.get_new_inputs(remapper=update_obsolete_vars)
+            _inputs = self._get_new_inputs(remapper=update_obsolete_vars)
```
Comment (hsaunders1904) [inline bluemira/codes/process/setup.py:107]: Same here

**Review by hsaunders1904 — COMMENTED** [review]
Looks fine to me, just one thing I noticed linking back to #748

File: bluemira/codes/plasmod/api.py (hunk 3 above)
Comment (je-cook) [reply bluemira/codes/plasmod/api.py:346]: Oops yeah bad rebase

**Review by hsaunders1904 — APPROVED** [review]

**Pull request merged**